exec/java-exec/src/main/java/org/apache/drill/exec/planner/common/DrillAggregateRelBase.java

```diff
@@ -34,12 +34,17 @@
 
 import java.util.List;
 
-
 /**
  * Base class for logical and physical Aggregations implemented in Drill
  */
 public abstract class DrillAggregateRelBase extends Aggregate implements DrillRelNode {
 
+  public DrillAggregateRelBase(RelOptCluster cluster, RelTraitSet traits, RelNode child,
+                               ImmutableBitSet groupSet, List<ImmutableBitSet> groupSets, List<AggregateCall> aggCalls) {
+    super(cluster, traits, child, groupSet, groupSets, aggCalls);
+  }
+
+  @Deprecated // Will be removed before Calcite 2.0
```

Comment:
Since you have marked this constructor deprecated, it would be good to remove its usages if it wasn't done.

Also, please update the comment, we can remove this constructor independently on the Calcite version we use.